src/codegen_shared.h

```diff
@@ -6,11 +6,10 @@
 #include <llvm/IR/DebugLoc.h>
 #include <llvm/IR/IRBuilder.h>
 #include <llvm/IR/MDBuilder.h>
+#include "julia.h"
 
 #define STR(csym)           #csym
 #define XSTR(csym)          STR(csym)
-#include "julia.h"
-#include "llvm-version.h"
```

Comment:
So this is no longer needed since now the files that `include` it are expected to already have `include`d `llvm-version`?

Reply:
yes, that is how we generally expect these headers to be used